monkey/infection_monkey/post_breach/job_scheduling/linux_job_scheduling.py

```diff
@@ -0,0 +1,12 @@
+TEMP_CRON = "$HOME/monkey-schedule-jobs"
+
+
+def get_linux_commands_to_schedule_jobs():
+    return [
+        f'touch {TEMP_CRON} &&',
+        f'crontab -l > {TEMP_CRON} &&',
+        'echo \"# Successfully scheduled a job using crontab\" |',
+        f'tee -a {TEMP_CRON} &&',
+        f'crontab {TEMP_CRON} &&',
+        f'rm {TEMP_CRON}'
+    ]
```

Comment:
We should `rm TEMP_CRON` even if the previous commands fail.

Reply:
Done; should I merge now?